imgui-SFML.cpp

```diff
@@ -231,7 +231,7 @@ struct WindowContext {
         lastCursor = ImGuiMouseCursor_COUNT;
 
         joystickId = getConnectedJoystickId();
-        for (int i = 0; i < sf::Joystick::ButtonCount; ++i) {
+        for (int i = 0; i < static_cast<int>(sf::Joystick::ButtonCount); ++i) {
```

Comment:
Looks like an SFML/SFML bug we ought to fix. We shouldn't use a signed int for a count like this. But that's outside the scope of this PR.